package.json

```diff
@@ -1,101 +1,31 @@
-    "lint": "eslint --max-warnings 0 --config .eslintrc .",
-    "lint:fix": "eslint --max-warnings 0 --config .eslintrc . --fix",
-    "generate:readme:toc": "markdown-toc -i \"README.md\"",
-    "generate:assets": "npm run build && npm run generate:readme:toc",
-    "bump:version": "npm --no-git-tag-version --allow-same-version version $VERSION",
-    "prepublishOnly": "npm run generate:assets"
-  },
-  "dependencies": {
-    "@asyncapi/specs": "^6.7.1",
-    "@openapi-contrib/openapi-schema-to-json-schema": "~3.2.0",
-    "@stoplight/json": "^3.20.2",
-    "@stoplight/json-ref-readers": "^1.2.2",
-    "@stoplight/json-ref-resolver": "^3.1.5",
-    "@stoplight/spectral-core": "^1.16.1",
-    "@stoplight/spectral-functions": "^1.7.2",
-    "@stoplight/spectral-parsers": "^1.0.2",
-    "@stoplight/spectral-ref-resolver": "^1.0.3",
-    "@stoplight/types": "^13.12.0",
-    "@types/json-schema": "^7.0.11",
-    "@types/urijs": "^1.19.19",
-    "ajv": "^8.11.0",
-    "ajv-errors": "^3.0.0",
-    "ajv-formats": "^2.1.1",
-    "avsc": "^5.7.5",
-    "js-yaml": "^4.1.0",
-    "jsonpath-plus": "^7.2.0",
-    "node-fetch": "2.6.7"
+    "test": "turbo run test",
+    "lint": "turbo run lint",
+    "lint:fix": "turbo run lint:fix",
+    "build": "turbo run build",
+    "parser:test:unit": "turbo run test:unit --filter=@asyncapi/parser",
+    "parser:test": "turbo run test --filter=@asyncapi/parser",
+    "parser:test:browser": "turbo run test:browser --filter=@asyncapi/parser",
+    "parser:build": "turbo run build --filter=@asyncapi/parser",
+    "changeset": "changeset",
+    "version-packages": "changeset version",
+    "publish-packages": "turbo run build && changeset publish"
   },
   "devDependencies": {
-    "@asyncapi/avro-schema-parser": "^3.0.24",
-    "@jest/types": "^29.0.2",
-    "@swc/core": "^1.2.248",
-    "@swc/jest": "^0.2.22",
-    "@types/jest": "^27.4.1",
-    "@types/js-yaml": "^4.0.5",
-    "@types/node": "^18.16.1",
-    "@types/node-fetch": "^2.6.2",
-    "@typescript-eslint/eslint-plugin": "^5.36.2",
-    "@typescript-eslint/parser": "^5.36.2",
-    "browserify": "^16.3.0",
-    "browserify-shim": "^3.8.14",
-    "cross-env": "^7.0.3",
-    "eslint": "^8.23.0",
-    "eslint-plugin-github": "^4.3.7",
-    "eslint-plugin-security": "^1.5.0",
-    "eslint-plugin-sonarjs": "^0.15.0",
-    "jest": "^29.0.2",
-    "markdown-toc": "^1.2.0",
-    "path-browserify": "^1.0.1",
-    "puppeteer": "^17.1.1",
-    "ts-loader": "^9.3.1",
-    "ts-node": "^10.9.1",
-    "typescript": "^4.8.2",
-    "webpack": "^5.74.0",
-    "webpack-bundle-analyzer": "^4.6.1",
-    "webpack-cli": "^4.10.0"
+    "turbo": "1.13.3"
   },
-  "browserify": {
-    "transform": [
-      "browserify-shim"
-    ]
+  "engines": {
+    "node": ">=18"
   },
-  "browserify-shim": {
-    "node-fetch": "global:fetch"
+  "packageManager": "npm@10.2.4",
+  "workspaces": [
+    "apps/*",
+    "packages/*"
```

Comment:
We don't have this dir in this repo. Where does it come from?

Reply:
But we do not have such a directory, at least not yet. In fact, I believe both parser-js and multi-parser should be in `packages` dir instead of `app` right? 

This is the recommendation afaik https://turbo.build/repo/docs/crafting-your-repository/structuring-a-repository#declaring-directories-for-packages

Reply:
> But we do not have such a directory, at least not yet. In fact, I believe both parser-js and multi-parser should be in `packages` dir instead of `app` right?
> 
> This is the recommendation afaik https://turbo.build/repo/docs/crafting-your-repository/structuring-a-repository#declaring-directories-for-packages

Actually i think they should be app only generally the packages contain the code that is common. here in this example the packages is used just for explaining the how the structure of files should be. I think same logic can be applied to apps also. Turborepo doesnt treats apps and packages differently they are same for it.

Reply:
> here in the example(that you have shared) the packages is used just for explaining the how the structure of files should be.

I don't see this explained anywhere 🤔 I don't think it is what they are mentioning:

> We recommend starting with splitting your packages into apps/ for applications and services and packages/ for everything else, like libraries and tooling.

Parser-JS is neither an app or a service but a library and tool right?

Reply:
ok so should i shift from parser-js to packages ??

Reply:
ok moved from apps to packages.